core/pta/bcm/hw_rng.c

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2019, Broadcom
+ */
+
+#include <drivers/bcm_hwrng.h>
+#include <io.h>
+#include <kernel/pseudo_ta.h>
+#include <trace.h>
+
+#define HW_RNG_SERVICE_UUID \
+		{ 0x6272636D, 0x2019, 0x0201,  \
+		{ 0x42, 0x43, 0x4D, 0x5F, 0x52, 0x4E, 0x47, 0x30 } }
+
+#define PTA_BCM_HW_RNG_CMD_GET	0
```

Comment:
Please document the parameters, see `lib/libutee/include/pta_system.h` for examples.

Reply:
Ok